subtrack.MAUI/Services/SubscriptionsCalculator.cs

```diff
@@ -10,4 +10,13 @@ public static decimal GetMonthlyCost(this IEnumerable<Subscription> subscription
         var price = subscriptions.Sum(x => x.Cost);
         return price;
     }
+
+    public static int GetDueDays(this Subscription subscription)
```

Comment:
Add some unit tests for this, lmk if you need any help.

Some cases to cover
- GetDueDays_DueDateHasPassed_returns_negative_duedays
- GetDueDays_returns_correctduedate (this can be a theory so we test multiple different inputs)
- GetDueDays_nullvalue_throws_argumentnullexception